crates/jit/src/function_table.rs

```diff
@@ -2,13 +2,15 @@
 //!
 //! This module is primarily used to track JIT functions on Windows for stack walking and unwind.
 
+type FunctionTableReloc = wasmtime_environ::CompiledFunctionUnwindInfoReloc;
+
 /// Represents a runtime function table.
 ///
 /// The runtime function table is not implemented for non-Windows target platforms.
-#[cfg(not(target_os = "windows"))]
+#[cfg(not(any(target_os = "windows", target_os = "macos", target_os = "linux")))]
 pub(crate) struct FunctionTable;
 
-#[cfg(not(target_os = "windows"))]
+#[cfg(not(any(target_os = "windows", target_os = "macos", target_os = "linux")))]
```

Comment:
Since wasmtime only compiles on these platforms right now, could this `#[cfg]` be removed?